phygnn/phygnn.py

```diff
@@ -142,8 +182,37 @@ def _preflight_p_fun(self, x, y_true, p, p_kwargs):
 
         logger.debug('p_fun passed preflight check.')
 
-    def _preflight_data(self, x, y, p):
-        """Run simple preflight checks on data shapes."""
+    def preflight_data(self, x, y, p):

```

Comment:
Why'd you decided to "expose" these methods?

Reply:
People using this are _probably_ fairly competent and should be able to preflight their data before running training or predictions